src/index.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import PropTypes from 'prop-types';
+import App from './components/App';
+// import PropTypes from 'prop-types';
```

Comment:
Consider deleting commented out codes. If you need to review a previous version of your code, use git history.